containers/jml-extract-lambda/config.json

```
@@ -1,6 +1,6 @@
 {
   "name": "jml-extract-lambda",
-  "version": "1.3.3",
+  "version": "1.4.0",
```

```suggestion
  "version": "1.3.4",
```
The updates to the requirements are all patches, so I make this a patch update also

---

amended